routes/add_event.js

```diff
@@ -26,13 +28,7 @@ module.exports = (req, res) => {
     if (error) {
       res.send(error);
     } else {
-      if (result.statusCode >= 400 && result.statusCode <= 499) {
-        res.send(`error :( ${result.body.reasons}`);
-      } else if (result.statusCode >= 500 && result.statusCode <= 599) {
-        res.send('server error');
-      } else if (result.statusCode >= 200 && result.statusCode <= 299) {
-        res.redirect('/');
-      }
+      res.redirect(`/event/${req.locals.id}`);
```

Comment:
Route is broken - should be `events` and you need to add `body` as a parameter on line 27 in order to access the id of the new event `body._id`